scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -213,14 +240,15 @@ class RunCoverage(
     }
   }
 
-  private fun combineCoverageReports(coverageResultList: List<CoverageReport>):
-    CoverageReportContainer {
-      val containerBuilder = CoverageReportContainer.newBuilder()
-      coverageResultList.forEach { report ->
-        containerBuilder.addCoverageReport(report)
-      }
-      return containerBuilder.build()
+  private fun combineCoverageReports(
+    coverageResultList: List<CoverageReport>
+  ): CoverageReportContainer {
+    val containerBuilder = CoverageReportContainer.newBuilder()
+    coverageResultList.forEach { report ->
+      containerBuilder.addCoverageReport(report)
     }
+    return containerBuilder.build()
+  }
```

Comment:
```suggestion
    return CoverageReportContainer.newBuilder().apply {
      addAllCoverageReport(coverageResultList)
    }.build()
  }
```

Another approach (optional).

Reply:
Thanks, updated with the same.